.github/workflows/ci.yml

```diff
@@ -707,6 +709,30 @@ jobs:
             -Dtest.redshift.aws.region="${AWS_REGION}" \
             -Dtest.redshift.aws.access-key="${AWS_ACCESS_KEY_ID}" \
             -Dtest.redshift.aws.secret-key="${AWS_SECRET_ACCESS_KEY}"
+      - name: Fault Tolerant Redshift Tests
```

Comment:
Why do we need a separated job instead of reusing to the existing `cloud-tests` like BigQuery connector?

Reply:
This starts and stops another redshift cluster. 

When multiple concurrent tests run against the same redshift cluster, tests slow down and this can result in timeouts.